### PR TITLE
Draft: use ansible to create security group

### LINKED ISSUE #9: Refactoring: provisioning tool instead of bash scripts
It can be one of the two: Terraform or Ansible

### PR DESCRIPTION
**Pull request by yerdreic**

### CODE REVIEW

File: ansible/ec2_config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# - name: "Set variables"
```
Comment (lmilbaum) [inline ansible/ec2_config.yml:1]: Remove the comments. Your decision to go with a vars file is the preferred one.

File: ansible/ec2_config.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  hosts: localhost
+  connection: local
+  vars_files:
+    - variables.yaml
+  tasks:
+    - name: create security group
+      amazon.aws.ec2_group:
+        name: "{{ sg_name }}"
+        description: "{{ sg_description }}"
+        profile: "{{ aws_profile }}"
```
Comment (lmilbaum) [inline ansible/ec2_config.yml:29]: What is this key for?

**Comment by lmilbaum** [comment]
I am aware that this is a draft PR. Just in case, please add to the README the instructions.

File: ansible/ec2_config.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#     profile_name: "PocProfile"
+---
+- name: ec2 config
+  hosts: localhost
+  connection: local
+  vars_files:
+    - variables.yaml
+  tasks:
+    - name: create security group
+      amazon.aws.ec2_group:
```
Comment (lmilbaum) [inline ansible/ec2_config.yml:26]: The module name should be: `amazon.aws.ec2_security_group`

File: ansible/ec2_config.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        name: "{{ sg_name }}"
+        description: "{{ sg_description }}"
+        profile: "{{ aws_profile }}"
+        region: "{{ aws_region }}"
```
Comment (lmilbaum) [inline ansible/ec2_config.yml:30]: vpc_id is missing

**Comment by lmilbaum** [comment]
And, please add a play for removing the resources

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Comment by yerdreic** [comment]
Not relevant anymore

**Pull request closed without merging**